roles/ingress_nginx/defaults/main.yml

```diff
@@ -24,9 +24,9 @@ ingress_nginx_release_defaults:
     config:
       annotations-risk-level: Critical
     metrics:
-      enabled: true
+      enabled: "{{ deploy_prometheus_stack }}"
```

Comment:
This should be something like this into this defaults file:

ingress_ngnix_pometheus_stack_enabled: true